File: modules/cli/src/test/resources/golden/input/entities/Groesse.scala
package com.breuninger.entdecken.domain.model

import java.util.Locale

case class Groesse(
    id: String,
    dimsortierung: Map[Locale, Int],
    groessenanzeige: Map[Locale, String],
    groessenfilter: Map[Locale, String]
)

File: modules/cli/src/test/resources/golden/expected/entities/Groesse.kt
package com.breuninger.entdecken.domain.model

import java.util.Locale

data class Groesse(
    val id: String,
    val dimsortierung: Map<Locale, Int>,
    val groessenanzeige: Map<Locale, String>,
    val groessenfilter: Map<Locale, String>
)
